### PR TITLE
Storage improve coverage

### PR DESCRIPTION
**Pull request by pavelkrolevets**
* github.com/bloxapp/ssv/storage/kv	coverage: 78.9% of statements

### CODE REVIEW

File: storage/kv/badger_test.go [hunk 1]
```diff
@@ -167,6 +172,45 @@ func TestBadgerDb_SetMany(t *testing.T) {
...
 
+func TestBadgerDb_CountPrefix(t *testing.T) {
+	logger := logging.TestLogger(t)
+	db, err := NewInMemory(logger, basedb.Options{})
+	require.NoError(t, err)
+	defer db.Close()
+
+	prefix := []byte("prefix")
+	var i uint64
+	for i = 0; i < 100; i++ {
```
Comment (nkryuchkov) [inline storage/kv/badger_test.go:183]:
```suggestion
	for i := uint64(0); i < 100; i++ {
```

File: storage/kv/badger_test.go [hunk 2]
```diff
@@ -141,6 +141,11 @@ func TestBadgerDb_GetMany(t *testing.T) {
 	})
 	require.NoError(t, err)
 	require.Equal(t, 4, len(results))
+	err = db.GetMany(prefix, [][]byte{}, func(obj basedb.Obj) error {
```
Comment (nkryuchkov) [inline storage/kv/badger_test.go:144]: This would be a bit more readable with a newline
```suggestion
	
	err = db.GetMany(prefix, [][]byte{}, func(obj basedb.Obj) error {
```

File: storage/kv/badger_test.go [hunk 3]
```diff
@@ -167,6 +172,45 @@ func TestBadgerDb_SetMany(t *testing.T) {
...
+	logger := logging.TestLogger(t)
+	db, err := NewInMemory(logger, basedb.Options{})
+	require.NoError(t, err)
+	defer db.Close()
+
+	prefix := []byte("prefix")
+	key := []byte("key")
+	value := []byte("value")
+	err = db.Set(prefix, key, value)
+	require.NoError(t, err)
```
Comment (nkryuchkov) [inline storage/kv/badger_test.go:202]:
```suggestion
	require.NoError(t, err)
	
```

File: storage/kv/badger_test.go [hunk 4]
```diff
@@ -167,6 +172,45 @@ func TestBadgerDb_SetMany(t *testing.T) {
...
+	newValue := []byte("newValue")
+	tx := func(txn basedb.Txn) error {
+		return db.Set(prefix, key, newValue)
+	}
+	err = db.Update(tx)
+	require.NoError(t, err)
```
Comment (nkryuchkov) [inline storage/kv/badger_test.go:208]:
```suggestion
	require.NoError(t, err)
	
```

File: storage/kv/txn_test.go [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+	defer cancel()
+
+	zapCore, _ := observer.New(zap.DebugLevel)
+	logger := zap.New(zapCore)
+	options := basedb.Options{
+		Reporting: true,
+		Ctx:       ctx,
+	}
+
+	db, err := NewInMemory(logger, options)
```
Comment (nkryuchkov) [inline storage/kv/txn_test.go:28]: it's a bit hard to follow the logic without splitting code blocks with newlines

File: storage/kv/txn_test.go [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+		all = append(all, obj)
+		return nil
+	})
+	require.Equal(t, 100, len(all))
+	require.NoError(t, err)
+	visited := map[string][]byte{}
+	for _, item := range all {
+		visited[string(item.Key)] = item.Value
+	}
+	require.Equal(t, 100, len(visited))
```
Comment (y0sher) [inline storage/kv/txn_test.go:162]: I would rather we check all the values instead just the length

**Review by y0sher — CHANGES_REQUESTED** [review]
minor addition

**Comment by kchojn** [comment]
Closed by: https://github.com/ssvlabs/ssv/pull/2094
